config/routes.rb

```diff
@@ -4,6 +4,16 @@
   devise_scope :user do
     get '/users/sign_out' => 'devise/sessions#destroy'
   end
+  
+  namespace :api, default: {format: :json} do 
+    namespace :v1,  default: {format: :json} do
+      resources :users, only: [:index, :show] do
+      resources :posts, only: [:index, :show, :create] do
+        resources :comments, only: [:index, :create]
+      end
+    end
+  end
+end
```

Comment:
- [ ] Kindly add the endpoint for `signing up or registering a new user` and the endpoint for `log in`.

Reply:
Hello @andersonlebon ,
By default authentication is handled by devise, this extends to the API endpoints as well so we do not need to add auth endpoints as devise already handles this for us. But, I tried to expose it. 
Kindly, take into consideration my concern.
 Thank you for your review.